README.md

```diff
@@ -35,21 +35,41 @@ And more to come, see Marker's goals and limitations below.
 
 The following is an abbreviated guide. Check out [The Marker Book] for detailed instructions and additional information.
 
-[The Marker Book]: rust-marker.github.io/marker/book
+[The Marker Book]: https://rust-marker.github.io/marker/book
 
 ### Installation
 
+#### Download pre-compiled binaries (recommended)
```

Comment:
I may leave these sections commented out for now, but don't forget to uncomment them right before the 0.3.0 release

Reply:
I was thinking of commenting out these sections, but that looks quite ugly because you can't nest comments in markdown. Anyway, I'll move them to a separate PR

Reply:
Actually, making a PR doesn't make sense right now before this PR is merged, especially because there will be a merge commit created.

You can use `git revert 0f17cd9` before the release to revert the commit that removes the docs for the pre-compiled binaries

Reply:
Sounds like a perfect solution! :D